### DISCUSSION TITLE
Exploring the future of web development with WebAssembly and PHP

### DISCUSSION THREAD

**Discussion by adamziel** [WordPress/developer-blog-content#135] [Topic Ideas]
Since Playground and WebAssembly are only going to more important, the https://developer.wordpress.org/news/ audience could be interested in an article exploring the current state and future implications of this technology.

I co-authored the following article with Angel De Miguel and Rafael Fernández López from wasmlabs.dev. It is not published yet and we'd be happy to contribute it to developer.wordpress.org/news/ – what do you think?

[Exploring the future of web development with WebAssembly and PHP](https://docs.google.com/document/d/1NlPLW1V6GzgHz3tWV0MWTL1_7w_Qy3pgNaJw_XEQtdg/edit#heading=h.q2wjhw6qz0fm)

**Comment by adamziel** [comment]
cc @justintadlock @bph @annezazu @mburridge @ndiego

**Reply by annezazu** [reply]
I love it and agree!

**Reply by justintadlock** [reply]
Catching up (was on vacation). This looks great, @adamziel.

**Comment by mburridge** [comment]
LGTM 👌

**Comment by adamziel** [comment]
Yay, I’m glad to see the good feedback! What would be the next steps here?

**Comment by bph** [comment]
Me, too @adamziel 
Next step:  The topic idea will be discussed in the Editorial Group meeting on July 6, 2023. It might get approved as is or more discussion will ensue :-)

**Comment by Angelmmiguel** [comment]
Hello! 👋 

I'm Angel, part of the Wasm Labs team. Happy to help on any required changed / revision of the current article 😄.

Thank you!

**Comment by bph** [comment]
The topic has been approved. The discussion is now locked, 
Please continue the conversation on the issue
- #143

**Comment by bph** [comment]
Published: [Exploring the future of web development with WebAssembly and PHP](https://developer.wordpress.org/news/2023/08/exploring-the-future-of-web-development-with-webassembly-and-php/)
